schema/2.0.0/corpora_schema.md

```diff
@@ -145,13 +145,13 @@ The following ontology dependencies are *pinned* for this version of the schema.
 
 cellxgene requires ENSEMBL gene identifiers to ensure that all datasets it stores measure the same features and can therefore be integrated.
 
-The following gene annotation dependencies are *pinned* for this version of the schema.
+The following gene annotation dependencies are *pinned* for this version of the schema. **Cells from any organism are allowed as long as orthologs from these annotations are used.**
```

Comment:
Orthologs will not be a common use case. While it's an important note in the context of this update, I don't think it should be bolded because it will be relevant to a small subset of users. 

The _last_ thing we want is to encourage e.g. the fly community to map all their data to human orthologs so that they can publish on our platform. We would much prefer to add support for fly in a future update. These changes are only supported to enable cross species integration datasets. Jonah and @jahilton will need to be cautious about this. 

I suggest: 

```suggestion
The following gene annotation dependencies are *pinned* for this version of the schema. For multi-species experiments, cells from any organism are allowed as long as orthologs from the following species annotations are used.
```

Reply:
For consistency with the metadata fields, I always use organism and not species.

Reply:
Changed